prbt_default_application/launch/my_application.launch

```diff
@@ -57,10 +57,15 @@ limitations under the License.
       <arg name="load_robot_description" value="false" />
       <arg name="gripper" value="$(arg gripper)" />
 
-      <!-- Safe-torque-off-input from the safety controller is read to brake the robot before power-off
-           The signal is read via modbus and you can configure the modbus server ip -->
-      <!-- <arg name="sto" value="pnoz" /> -->
-      <!-- <arg name="sto_modbus_server_ip" value="192.168.0.10" /> -->
+      <!-- By default, the safety HW is configured to be PSS4000. If you want to used PNOZ or a different modbus
```

Comment:
I suggest adding (via comment) the following information:
- For each safety controller we only support a certain parameter configuration. We strongly recommend using our default configurations for the parameters for each safety controller.
- Switching the safety controller, the user should not forget to change the parameters accordingly.
-  Where can the default parameter configurations for each safety controller be found?

Reply:
added this configs in https://github.com/PilzDE/pilz_robots/blob/melodic-devel/prbt_hardware_support/README.md